src/main/scala/firrtl/stage/transforms/UpdateAnnotations.scala

```diff
@@ -22,8 +22,8 @@ class UpdateAnnotations(val underlying: Transform) extends Transform with Wrappe
 
     logger.info(s"Form: ${result.form}")
 
-    logger.debug(s"Annotations:")
-    remappedAnnotations.foreach( a => logger.debug(a.serialize) )
+    logger.trace(s"Annotations:")
+    logger.trace(JsonProtocol.serialize(remappedAnnotations))
 
```

Comment:
The code in this file is basically duplicated from `Compiler.scala`, @seldridge do you think it would be reasonable to move these into some common internal utility?